src/newspack-ui/scss/elements/misc/_reader-auth.scss

```
@@ -1,8 +1,7 @@
 // Re-adds some modal-like spacing for when the sign up form ends up embedded in a page.
-// Ignore a tag buttons as they will never be first..
 .newspack-ui {
 	&.newspack-reader-auth {
-		.newspack-ui__button:not(:first-child, a.newspack-ui__button):first-of-type {
+		.newspack-ui__button:not(:first-child):first-of-type {
```

```suggestion
		.newspack-ui__button:not(:first-child, a.newspack-ui__button):first-of-type {
```

This just affects the modal, not the block, and should be changed back -- see https://github.com/Automattic/newspack-plugin/pull/3267.